### DISCUSSION TITLE
Link Validation

### DISCUSSION THREAD

**Discussion by WeirDave** [phcode-dev/phoenix#1745] [Ideas]
It would be great to have an HTML Link validation "debug" feature that would find broken links before publishing.

**Comment by abose** [comment]
Can you elaborate with a example.

**Reply by WeirDave** [reply]
Certainly you have a project of  a website with several pages and those pages have multiple links to other pages. It would a selected tool that would check all the html links so see if there is a valid page for each of them.

**Reply by abose** [reply]
Got it, that's a nice feature to have!
